test/date_types.test.js

```diff
@@ -95,7 +97,19 @@ describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with local TZ', funct
       },
     }, function(err, found) {
       assert.ok(!err);
-      assert.ok(found);
+      assert.equal(found.id, 1);
+      done();
+    });
+  });
+
+  it('should find model instance by date field passing string', function(done) {
```

Comment:
It is a question how to pass where parameter to DATE field in different timezone via REST interface. 

f.e. server is in GMT-6000 timezone and client is in GMT+2000 timezone. on client side we do:  var filter = {where:{dateFiled:new Date(2016,11,22)}};  it will pass as {"where":{"dateFiled":"2016-12-21T22:00:00.000Z"}}

and dao will convert it to date "Wed Dec 21 2016 16:00:00 GMT-0600 (CST)" and pass to SQL as "2016-12-21"